cs/nuget/scripts/10-install-stack.ps1

```diff
@@ -0,0 +1,24 @@
+[CmdletBinding()]
+param ( )
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+Add-Type -AssemblyName 'System.IO.Compression.FileSystem'
+
+$stackInstaller = "$env:temp\stack-$(New-Guid).zip"
```

Comment:
Can we use the same random temp file name generation primitives that we use for [installing Boost](https://github.com/Microsoft/bond/blob/73f1861e0ae4b92822d78f0a013057dae84b5d1e/tools/ci-scripts/windows/Install-Boost.ps1#L76-L78) so we solve the same problem the same way, even in two different places.